From: Niklas Haas <git@haasn.dev>
Date: Sun, 27 Jun 2021 14:01:07 +0200
Subject: tests/libav: explicitly test only LE variants

Since our libav helpers rejects BE formats, using the NE-dependent
aliases here is wrong. This makes the test framework pass on BE
platforms. Though I'm not very convinced that the code actually works
correctly, it's probably impossible to test in the absence of BE GPUs to
test against.

Closes https://code.videolan.org/videolan/libplacebo/-/issues/160
---
 src/tests/libav.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/tests/libav.c b/src/tests/libav.c
index 183a4d0..d98609d 100644
--- a/src/tests/libav.c
+++ b/src/tests/libav.c
@@ -85,8 +85,8 @@ int main()
         }
     };
 
-    TEST(AV_PIX_FMT_YUV420P10, yuvp16);
-    TEST(AV_PIX_FMT_YUV420P16, yuvp16);
+    TEST(AV_PIX_FMT_YUV420P10LE, yuvp16);
+    TEST(AV_PIX_FMT_YUV420P16LE, yuvp16);
 
     static const struct pl_plane_data nv12[] = {
         {
@@ -134,8 +134,8 @@ int main()
         }
     };
 
-    TEST(AV_PIX_FMT_P010, p016);
-    TEST(AV_PIX_FMT_P016, p016);
+    TEST(AV_PIX_FMT_P010LE, p016);
+    TEST(AV_PIX_FMT_P016LE, p016);
 
     // Packed formats
     static const struct pl_plane_data r8[] = {
@@ -258,7 +258,7 @@ int main()
         }
     };
 
-    TEST(AV_PIX_FMT_GRAY16, r16);
+    TEST(AV_PIX_FMT_GRAY16LE, r16);
 
     static const struct pl_plane_data rgb16[] = {
         {
@@ -269,7 +269,7 @@ int main()
         }
     };
 
-    TEST(AV_PIX_FMT_RGB48, rgb16);
+    TEST(AV_PIX_FMT_RGB48LE, rgb16);
 
     static const struct pl_plane_data rgb565[] = {
         {
@@ -280,7 +280,7 @@ int main()
         }
     };
 
-    TEST(AV_PIX_FMT_RGB565, rgb565);
+    TEST(AV_PIX_FMT_RGB565LE, rgb565);
 
     // Test pl_frame <- AVFrame bridge
     struct pl_frame image;
